lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -66,7 +66,9 @@ defmodule Mix.Tasks.Release.Init do
     #   export ELIXIR_ERL_OPTIONS="-heart"
     # fi
 
-    # Set the release to work across nodes
+    # Set the release to work across nodes. If using a long name like the
```

Comment:
"my_app@127.0.0.1" is pretty short 😛 

What do you think about?

```suggestion
    # Set the release to work across nodes. If using a long name format like the
```